### PR TITLE
Final update to include AAA SC for Drupal 9

### PR DESCRIPTION
**Pull request by mgifford**
This includes the final SC for the Drupal example.

Realized that I couldn't bring over the notes for the report. Will have to follow up in the issue queue.

### CODE REVIEW

**Comment by dmundra** [comment]
@mgifford would you like me to add AAA to the catalog and schema as well like #82?

File: opat/drupal-9.yaml [hunk 1]
```diff
@@ -618,5 +618,132 @@ chapters:
...
+            adherence:
+              level: "supports"
+              notes: The web front-end is not a barrier to keyboard only users.
+      - num: "2.2.3"
+        components:
+          - name: "web"
+            adherence:
+              level: "supports"
+              notes: There are no timeouts in Drupal Core that would affect people with disabilities.
+      - num: "2.2.4
```
Comment (dmundra) [inline opat/drupal-9.yaml:679]: Missing closing double-quote here.

**Review by dmundra — CHANGES_REQUESTED** [review]
Found one minor typo and left you a previous comment about adding AAA to the catalog and schema.

**Comment by mgifford** [comment]
> @mgifford would you like me to add AAA to the catalog and schema as well like #82?

Yes, please. I think I see what you did but about to sign out for tonight.

File: opat/drupal-9.yaml [hunk 2]
```diff
@@ -618,5 +619,133 @@ chapters:
...
+          - name: "web"
+            adherence:
+              level: "supports"
+              notes: A front-end and back-end theme could be configured to allow this to comply.
+      - num: "1.4.7"
+        components:
+          - name: "web"
+            adherence:
+              level: "not-applicable"
+              notes:
```
Comment (dmundra) [inline opat/drupal-9.yaml:656]: @mgifford looks like we have empty `notes` here.

**Review by dmundra — CHANGES_REQUESTED** [review]
> > @mgifford would you like me to add AAA to the catalog and schema as well like #82?
> 
> ~Yes, please. I think I see what you did but about to sign out for tonight.~
> 
> I think I've got it in the patch now. Not too difficult if I got it right.

Excellent. The schema is looking good. Do you also want to add AAA to the catalog?

File: opat/drupal-9.yaml (hunk 2 above)
Comment (mgifford) [reply opat/drupal-9.yaml]: Fixed.

Comment (mgifford) [reply opat/drupal-9.yaml]: And yes, if you could at it to the catalog that would be great.

**Review by dmundra — APPROVED** [review]
Thanks, @mgifford for adding AAA. Merging.

**Pull request merged**